saleor/graphql/account/bulk_mutations.py

```diff
@@ -0,0 +1,58 @@
+import graphene
+
+from ...account import models
+from ..core.mutations import ModelBulkDeleteMutation
+
+
+class UserBulkDelete(ModelBulkDeleteMutation):
+    class Arguments:
+        ids = graphene.List(
+            graphene.ID,
+            required=True,
+            description='List of sale IDs to delete.')
+
+    class Meta:
+        abstract = True
+
+    @classmethod
+    def clean_instance(cls, info, instance, errors):
+        user = info.context.user
+        if instance == user:
+            cls.add_error(
+                errors, 'id', 'You cannot delete your own account.')
+        elif instance.is_superuser:
+            cls.add_error(
+                errors, 'id', 'Only superuser can delete his own account.')
```

Comment:
Should it be ```Superuser's account cannot be delelted``` since only ```instance.is_superuser``` is checked?

Reply:
We could also say "Cannot delete this account" or something like that to not reveal that this ID represents a superuser account, does it make sense? It would actually apply also to deleting single users.

Reply:
I admit this has been copied from the normal (non bulk) delete. I planned to extract that validation to a separate function, so it can be used in both without copying like this.